.github/workflows/ci.yml

```diff
@@ -26,3 +26,9 @@ jobs:
       if: always()
       with:
         files: "*/target/*-reports/TEST*.xml"
+
+    - name: Keep build artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: ivy-archives
```

Comment:
I do not like the name ivy-archives. May be an option could be to reuse the repo name? (Should be possible with ${{ github.event.repository.name }})